appconfig.default.json

```diff
@@ -60,5 +60,59 @@
 
     "DEFAULT_IDENTIFIER": "mywallet",
     
-    "NULL": null // just here so we can have a trailing , on every line
+    "NULL": null, // just here so we can have a trailing , on every line
```

Comment:
minor nit - this is here so all the meaningful config values all have a `,` on each line. Maybe move this to the bottom of the config..

It's here because then you don't have to change two lines if you add a new config value to the bottom.. it's only minor, but just pointing out it's purpose :)